### PR TITLE
eclean: Make cleaning invalid default

### PR DESCRIPTION
**Pull request by siddhanthrathod**

### CODE REVIEW

File: pym/gentoolkit/eclean/cli.py [hunk 1]
```diff
@@ -457,12 +458,12 @@ def optionSwitch(option, opts, action=None):
...
     getopt_options["short"]["distfiles"] = "fs:"
     getopt_options["long"]["distfiles"] = ["fetch-restricted", "size-limit="]
-    getopt_options["short"]["packages"] = "iNu"
+    getopt_options["short"]["packages"] = "iu"
     getopt_options["long"]["packages"] = [
         "ignore-failure",
         "changed-deps",
-        "clean-invalid",
         "unique-use",
+        "no-clean-invlaid",
```
Comment (thesamesam) [inline pym/gentoolkit/eclean/cli.py:466]: Typo!

File: pym/gentoolkit/eclean/cli.py [hunk 2]
```diff
@@ -478,7 +479,7 @@ def optionSwitch(option, opts, action=None):
     options["verbose"] = False
     options["changed-deps"] = False
     options["ignore-failure"] = False
-    options["clean-invalid"] = False
+    options["clean-invalid"] = True
```
Comment (thesamesam) [inline pym/gentoolkit/eclean/cli.py:482]: Should this instead be `no-clean-invalid` being False? Not sure, please check very carefully if the logic works correctly.

**Review by thesamesam — CHANGES_REQUESTED** [review]

File: pym/gentoolkit/eclean/cli.py (hunk 2 above)
Comment (siddhanthrathod) [reply pym/gentoolkit/eclean/cli.py:482]: If its False then it will be off by default no ?

Comment (thesamesam) [reply pym/gentoolkit/eclean/cli.py:482]: I just wanted to make sure that there wasn't confusion between `clean-invalid` and `no-clean-invalid` as options, if you get what I mean.

Comment (thesamesam) [reply pym/gentoolkit/eclean/cli.py:482]: (Really, we should replace the whole arg parsing with `argparse` or maybe `click`, but that's for another day.)

Comment (siddhanthrathod) [reply pym/gentoolkit/eclean/cli.py:482]: Ah you mean consistency okay okay I'll make it consistent

Comment (siddhanthrathod) [reply pym/gentoolkit/eclean/cli.py]: Better now ?

**Pull request closed without merging**